### ISSUE TITLE
Doesn't add button to forum.

### ISSUE THREAD

**Issue by epilimic** [live627/smf-steam-auth#1]
When I install this, I do not see the steam log in button. The installer said "Database installation complete!".

**Comment by live627** [comment]
Did you install it as a regular mod?

**Comment by epilimic** [comment]
I did two ways. Tried manually and also by creating a folder in the packages folder, uploading the files, then installing via the package manager. I wiped the smf install and started clean each method.

Installed mod list: http://i.imgur.com/vQke0JW.jpg
When I log out then click the login button: http://i.imgur.com/mWPTIw0.jpg

**Comment by live627** [comment]
ah, you need to fill in your Steam API key in the modification settings
area.

**Comment by epilimic** [comment]
Ok, did that. Now I'm getting this:

An Error Has Occurred!
The database value you're trying to insert does not exist: date_registered

**Comment by live627** [comment]
I fixed that in b7385e6dbd0ec9ad39cb80068b7271f0d5b54251 - please check.I

**Comment by epilimic** [comment]
Seems to be working. It made a new user for me. Only issue now that I can see is my avatar is showing a 404.

Not Found

The requested URL /avatars/https://steamcdn-a.akamaihd.net/steamcommunity/public/images/avatars/e6/e6ad11075f48fa156c8dadcc2b24a47e0fad2f6f_medium.jpg was not found on this server.

edit: what it's doing is it's taking the default avatar link and adding it in front of the supplied steam avatar link which breaks it. I've tried deleting the default avatar link from smf settings but it just defaults back to my main page url.

**Comment by epilimic** [comment]
Any idea?

**Comment by epilimic** [comment]
Alright so after some digging I was able to finally fix this. This may not be the right or best solution but it did the trick for me.

In both load.php and portalblocks.php there are numerous pieces of code dealing with avatars that looks like this:

'<img src="' . $modSettings['avatar_url'] . '/' . htmlspecialchars($row['avatar'])

What fixed it for me was removing the / from the middle in every instance. With the slash being there, it kept trying to load the avatar from my domain in front of the steamcdn server which in turn broke the url. Removing this slash makes the url work correctly. I'm not sure how many of the /'s I should have removed but I just took them all out and the avatars are loading up everywhere without issue. Hope this helps somebody.
